### ISSUE TITLE
[feature]: Adding event page

### ISSUE THREAD

**Issue by Aarti002** [garg3133/JagratiWebApp#204]
This page will provide the description  of events under jagrati .

**Comment by Aarti002** [comment]
hey @garg3133 please assign this to me will soon show you my progress .
Thank you!

**Comment by garg3133** [comment]
Sure @Aarti002, go ahead!

**Comment by tmo66** [comment]
I think @garg3133 Sir did a mistake here, #205 is related to issue #198. (Removing the linked PR)

**Comment by garg3133** [comment]
> I think @garg3133 Sir did a mistake here, #205 is related to issue #198. (Removing the linked PR)

Yeah, thanks!

**Comment by Aarti002** [comment]
Please give me some time i m working on it,actually i'm facing some
problems soon i will let you know.
Thank you

**Comment by coding-geek21** [comment]
can i work on this issue? @garg3133

**Comment by SmritiAgrawal04** [comment]
Is this issue being worked upon? If not, allow me to.

**Comment by garg3133** [comment]
Yes @coding-geek21, this issue is available to be taken. Would you like to work on this?

**Comment by coding-geek21** [comment]
> Yes @coding-geek21, this issue is available to be taken. Would you like to work on this?

Yes @garg3133  I would like to work on this

**Comment by garg3133** [comment]
Great @coding-geek21, assigning it to you.

Here are a few pages you can refer to while designing the page:
* https://sac.iiitdmj.ac.in/events/event/3/
* https://bandhuodisha.in/ankurayan/detail/2020/

or you may suggest some designs of your own. You can take a look at the models file of the `events` app for an idea of what all data we want to show on that page.

**Comment by coding-geek21** [comment]
@garg3133 Can i work on this as a part of LGM-SOC'21
